mroonga.rb

```diff
@@ -25,6 +27,14 @@ class Mroonga < Formula
     depends_on "cmake" => :build
     depends_on "boost" => :build
     depends_on "mysql"
+  elsif build.include?("use-homebrew-mysql56")
+    depends_on "cmake" => :build
+    depends_on "boost" => :build
```

Comment:
I appreciate your feedback. You are right. It successfully was built without `"boost"`.
I committed new code again, would you please check it again.